script.js

```diff
@@ -1,4 +1,98 @@
+var generateRandomInteger = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+
+  return diceNumber;
+};
+var totalGamePlayed = 0;
+var totalWin = 0;
+var totalLose = 0;
+var totalDraw = 0;
+var generateRandomPlay = function () {
+  number = generateRandomInteger();
+  if (number == 1) play = "stone";
+  if (number == 2) play = "paper";
+  if (number == 3) play = "scissor";
+  console.log(play);
+  return play;
+};
+var paperScissorStone = function (input) {
+  if (input !== "stone" && input !== "scissor" && input !== "paper") {
+    return `please input "scissor","paper", or "stone"`;
+  } else {
```

Comment:
great work implementing what we learnt in class about operators and if statements!